style.css

```diff
@@ -185,3 +182,389 @@ a:hover {
   order: 4;
   border-radius: 8px;
 }
+
+.works {
+  display: grid;
+  grid-template-columns: auto;
+  align-items: center;
+  width: 375px;
+  height: 3455px;
+  background: #fff;
+  order: 2;
+  position: absolute;
+  margin-top: 590px;
+  margin-left: 0%;
+  float: left;
+  top: 0;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 107px;
+  align-items: center;
+  justify-content: center;
+  justify-content: space-between;
+  left: 24px;
+}
+
+#recent {
+  width: 327px;
+  height: 10px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 30px;
+  align-items: center;
+  justify-content: center;
+  color: #1c1a19;
+}
+
+.line {
+  width: 327px;
+  margin-top: 24px;
+}
+
+.frame {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 3028px;
+  align-items: center;
+  gap: 24px;
+  position: relative;
+  bottom: 50px;
+}
+
+.featured {
+  justify-content: center;
+  gap: 24px;
+  width: 375px;
+  height: 568px;
+}
+
+.img {
+  width: 327px;
+  height: 237px;
+  background: url('../Images/Work-img1.png') no-repeat;
```

Comment:
[OPTIONAL] **There is an extra dot `.` in your image URL kindly make sure you keep one dot. 
Your code should look like this :arrow_down:**

` list-style-image: url("./Images/Rectangle55.png");`  **Notice that I removed one dot from the URL path**